src/components/Review/Applications/Actions.js

```diff
@@ -116,14 +116,14 @@ const RowActionsContainer = ({
 
 RowActionsContainer.propTypes = {
     row: RowPropTypes,
-    isCollapseOpen: PropTypes.bool.isRequired,
-    toggleCollapse: PropTypes.func.isRequired,
-    actionToConfirm: PropTypes.func.isRequired,
-    confirmAction: PropTypes.func.isRequired,
-    cancelAction: PropTypes.func.isRequired,
-    handleAction: PropTypes.func.isRequired,
-    handleApprove: PropTypes.func.isRequired,
-    executingAction: PropTypes.bool.isRequired,
+    isCollapseOpen: PropTypes.bool,     /* Removed isRequired from all props */
+    toggleCollapse: PropTypes.func,
+    actionToConfirm: PropTypes.func,
+    confirmAction: PropTypes.func,
+    cancelAction: PropTypes.func,
+    handleAction: PropTypes.func,
+    handleApprove: PropTypes.func,
+    executingAction: PropTypes.bool,
```

Comment:
These props refer to the actions of RowACtions from Review/Applications. They should not be reused as is.

If you want, you can create a more generic "RowActions*" component that both this and the new offers table use to create their custom actions, with their custom propTypes rules. Otherwise, create a similar component for the offers table feature.

Bottom line, these props must be required for now, don't change them just for reusability